### DISCUSSION TITLE
What happened to `jaegertracing/jaeger-opentelemetry-agent`?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [jaegertracing/jaeger#2849] [Q&A] [answered]
When we use with OpenTelemetry, before there is a docker-compose demo, which was mentioned in my [old ticket](https://github.com/jaegertracing/jaeger/issues/2407).

What happened to `jaegertracing/jaeger-opentelemetry-agent`? Is it depreciated?

If yes, is there a new way or any example to use with OpenTelemetry? Thank you!

**Comment by joe-elliott** [comment, marked as answer]
We are currently reworking the way that we use the OpenTelemetry pieces in Jaeger.  The goal is to have something like `jaegertracing/jaeger-opentelemetry-agent`, but it has currently been removed from the codebase.  The existing docker images should continue working for the foreseeable future as we haven't made changes that are not backwards compatible.

Another option would be to use the opentelemetry collector with the [Jaeger exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/jaegerexporter) that pushes into your collector layer.

**Reply by hongbo-miao** [reply]
Thanks Joe! I think this is the new way currently being experimented, right?
https://github.com/open-telemetry/opentelemetry-collector/tree/main/examples/demo
https://github.com/jaegertracing/jaeger-otelcol

**Reply by joe-elliott** [reply]
Yup, some discussion here: https://github.com/jaegertracing/jaeger/issues/2837 on how to integrate that into the main repo.
